components/fhir-core/src/main/java/org/wso2/healthcare/codegen/tool/framework/fhir/core/FHIRSpecParser.java

```diff
@@ -218,6 +137,97 @@ public void parse(ToolConfig toolConfig) {
         populateValues();
     }
 
+    public void parseIG(ToolConfig toolConfig, String igName, String igDirPath, FilenameFilter jsonFileFilter) {
```

Comment:
Let's take the fileFilter inside the method, no need to have as parameter since it will be json

Reply:
fixed in [e69346a](https://github.com/wso2/open-healthcare-codegen-tool-framework/pull/9/commits/e69346ab6e54f540c48d896e4c775dcad7223f05)